glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -63,12 +63,16 @@ internal interface LibGleanFFI : Library {
 
     fun glean_boolean_set(glean_handle: Long, metric_id: Long, value: Byte)
 
+    fun glean_boolean_should_record(glean_handle: Long, metric_id: Long): Byte
```

Comment:
Instead of increasing the FFI API surface, since this function always returns a `Byte` and has the same arguments, I just wish we had a way to pass in Enums through the FFI :|

Reply:
weeeeell, .... not impossible, just right now not worth it.